nntrainer/tensor/hgemm/hgemm_padding/hgemm_padding_a.h

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Sungsik Kong <ss.kong@samsung.com>
+ *
+ * @file   hgemm_padding_a.h
+ * @date   05 July 2024
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Sungsik Kong <ss.kong@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  This is a header file for padding function used in hgemm
+ *
+ */
+
+/**
+ * @brief Padding function for matrix A in HGEMM
+ *
+ * @param A src matrix to pad
+ * @param Ap dst matrix after padding
+ * @param M row length of matrix A
+ * @param K col length of matrix A
```

Comment:
```suggestion
 * @param M the number of rows of matrix A
 * @param K the number of cols of matrix A
```
Please consider to revise this comment.
The phrase "row length" may lead some confusions.

Reply:
Got it!